assets/capability/capabilities/homealarm_state.json

```diff
@@ -1,17 +1,17 @@
 {
   "type": "enum",
   "title": {
-    "en": "Home alarm state",
-    "nl": "Thuisalarm status",
-    "de": "Heim-Alarm-Status",
-    "fr": "Etat de l'alarme de maison",
-    "it": "Stato dell'allarme domestico",
-    "sv": "Hemlarmsstatus",
-    "no": "Hjemmealarmstatus",
-    "es": "Estado de la alarma doméstica",
-    "da": "Hjemmealarmstatus",
-    "ru": "Состояние домашней сигнализации",
-    "pl": "Stan alarmu domowego"
+    "en": "The home alarm is",
```

Comment:
This doesn't seem right.

Reply:
I am not qualified to assess what is the correct way to speak english, but in my view the "state" of something is a redundant word as it is implied when you refer to the actual state. That is, I believe this sounds best with:
"The home alarm is armed"
rather than
"The home alarm state is armed"

Nevertheless feel free to change it to whatever you like. I have already overridden the entire capability to improve this so I don't need this fix in the homey code at all, but I hope it can help others.